packages/binding-coap/src/coap-server.ts

```diff
@@ -197,6 +215,52 @@ export default class CoapServer implements ProtocolServer {
         });
     }
 
+    private formatCoreLinkFormatResources() {
+        return Array.from(this.coreResources.values())
+            .map((resource) => {
+                const formattedPath = `</${resource.urlPath}>`;
+                const parameters = Array.from(resource.parameters?.entries() ?? []);
+
+                const parameterValues = parameters.map((parameter) => {
+                    const key = parameter[0];
+                    const values = parameter[1].join(" ");
+                    return `${key}="${values}"`;
+                });
+
+                return [formattedPath, ...parameterValues].join(";");
+            })
+            .join("");
+    }
+
+    private handleWellKnownCore(req: IncomingMessage, res: OutgoingMessage) {
+        if (req.method === "GET") {
+            res.setOption("Content-Format", "application/link-format");
+            res.code = "2.05";
+            const payload = this.formatCoreLinkFormatResources();
+            res.end(payload);
+        } else {
+            res.code = "4.05";
+            res.end("Method Not Allowed");
+        }
+    }
+
+    private negotiateContentFormat(
+        req: IncomingMessage,
+        res: OutgoingMessage,
+        availableContentFormats: string[],
+        defaultContentFormat: string
+    ) {
+        const accept = req.headers.Accept;
+        console.log(accept);
```

Comment:
```suggestion
```

Reply:
Agree, remove or add proper logging

Reply:
Oh, thank you, that was only for debugging purposes 😅